src/hb-ot-math-table.hh

```diff
@@ -515,10 +515,10 @@ struct MathGlyphAssembly
     if (parts_count)
     {
       int64_t mult = font->dir_mult (direction);
-      hb_array_t<const MathGlyphPartRecord> arr = partRecords.sub_array (start_offset, parts_count);
-      unsigned int count = arr.length;
-      for (unsigned int i = 0; i < count; i++)
-	arr[i].extract (parts[i], mult, font);
+      + hb_enumerate (partRecords.sub_array (start_offset, parts_count))
```

Comment:
Why enumerate then array-lookup?!  Just zip it with `hb_array (parts, parts_count)`!

Reply:
Hehehe, nice, done